lib/fog/compute/google/requests/insert_server.rb

```diff
@@ -110,7 +110,7 @@ def handle_disks(options)
           disks
         end
 
-        def handle_networks(options)
+        def handle_networks(options, zone_name)
```

Comment:
1) We should avoid adding extra required parameters. Is it _really_ needed? I would suggest just objectifying the subnetwork resource properly. Subnetwork model is intimately aware of what it is and how it should behave, why not use it?
2) Why are we requiring zone name and then extracting region out of it?